circuit_maintenance_parser/processor.py

```diff
@@ -117,16 +117,27 @@ class CombinedProcessor(GenericProcessor):
 
     def process_hook(self, maintenances_extracted_data, maintenances_data):
         """All the parsers contribute with a subset of data that is extended."""
-        # We only expect one data object from these parsers
         if len(maintenances_extracted_data) == 1:
             self.combined_maintenance_data.update(maintenances_extracted_data[0])
         else:
-            raise ProcessorError(f"Unexpected data retrieved from parser: {maintenances_extracted_data}")
+            maintenances_data.extend(maintenances_extracted_data)
 
     def post_process_hook(self, maintenances_data):
-        """After processing all the parsers, we try to combine all the data together."""
+        """After processing all the parsers, we try to combine all the data together.
+
+        For some notifications there can be multiple maintenances in a single file. To handle this, maintenances are store in a
```

Comment:
maybe move this explanation to `process_hook` where this logic actually take place?